Sources/ApolloTestSupport/TestMockSelectionSetMapper.swift

```diff
@@ -0,0 +1,50 @@
+#if !COCOAPODS
+@testable import ApolloAPI
+#endif
+@testable import Apollo
```

Comment:
This needs to be moved up into the `#if !COCOAPODS` directive otherwise the cocoapods spec linting will fail with a warning that the file is already part of the `Apollo` module. [Part of what was fixed](https://github.com/apollographql/apollo-ios/pull/2937/files#diff-b3d3d639f815a2b3dcee78379825c377b4036869559a53c6eeb302aa1587fccfL3) for #2937.